bin/grocery-delivery

```diff
@@ -66,8 +66,15 @@ def read_checkpoint
 end
 
 def full_upload(knife)
-  GroceryDelivery::Log.warn('Uploading all cookbooks')
-  knife.cookbook_upload_all
+  if GroceryDelivery::Config.berks
+    GroceryDelivery::Log.warn('Uploading all cookbooks with berkshelf')
+    GroceryDelivery::Log.debug("using cookbook paths: 
```

Comment:
don't leave the newline in the string, do:

```
    GroceryDelivery::Log.debug(
      "using cookbook paths:  #{GroceryDelivery::Config.cookbook_paths.to_s}"
    )
```

(If you had to further break it, then our style says to use string concatenation to cross lines)